e-digi/src/br/com/caelum/editora/dto/LivroView.java

```diff
@@ -0,0 +1,70 @@
+package br.com.caelum.editora.dto;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+
+import br.com.caelum.editora.modelos.Autor;
+import br.com.caelum.editora.modelos.Categoria;
+import br.com.caelum.editora.modelos.Livro;
+
+public class LivroView {
```

Comment:
Se você já tem todas as informações de um Livro na classe Livro, por que precisa de uma nova classe com essas mesmas informações?

Reply:
Bem é que na classe Livro() o toString() retorna todos os atributos dele e nesse card não é pedido o Isbn.

Reply:
o toString é a única forma de elaborar uma string que imprima as informações do Objeto?

Reply:
Eu não entendi, eu não consego pensar em outra maneira de fazer isso sem utilizar os getters e sem o toString. Como eu faria isso ?

Reply:
Um método que parece o toString mas não é o toString.

Reply:
Você está sugerindo que eu crie e implemente um método assim, que retornaria uma String com as informações de pesquisa do Livro, dentro da própria classe? Algo mais ou menos assim:
`public String informacoesDePesquisa() {
		return "Título: " + this.titulo + " | Autor: " + this.autor + " | Ano de publicação: " + criadoEm + " | Edicação: " + this.edicao + " | Categoria: " + this.categoria + " | Preço" + this.preco + " | Número de páginas" + this.paginas + " | Resumo: " + this.resumo + " | Sumário: " + this.sumario;  
	}`

Reply:
Parece que sim

Reply:
Entendi